templates/catalog/brands.tpl

```diff
@@ -6,7 +6,7 @@
 
 {block name='content'}
   {block name='brand_header'}
-    <h1>{l s='Brands' d='Shop.Theme.Catalog'}</h1>
+    <h1 class="h1 mb-4">{l s='Brands' d='Shop.Theme.Catalog'}</h1>
```

Comment:
isn't it the job for `templates/components/section-title.tpl`? I may be wrong

Reply:
@NeOMakinG This component take care of the h2 tag, we use it for modules, and sections with products like ps_bestsellers etc.
I think we can also make a component deals with h1 tag.

Reply:
@NeOMakinG :)

Reply:
Oh sorry I did miss it, well I would make it reusable yes, this is not like we would add a complex reusable component, it's a simple one, let's go